ipa-core/src/ff/ec_prime_field.rs

```diff
@@ -125,15 +127,19 @@ impl From<Scalar> for Fp25519 {
     }
 }
 
-/// conversion from BA256 into Fp25519
+/// Conversion from BA256 into Fp25519
+///
+/// Values are reduced modulo the field order.
 impl From<BA256> for Fp25519 {
     fn from(s: BA256) -> Self {
         let mut buf: GenericArray<u8, U32> = [0u8; 32].into();
         s.serialize(&mut buf);
+        // Reduces mod order
         Fp25519::deserialize(&buf)
     }
 }
 
+/*
```

Comment:
Can we just delete this code instead?

Reply:
Yes, I meant to remove this -- unless @danielmasny has a reason why it should stay.

Reply:
I think I have only used this to debug. Since we do not use it, we can delete it. I think I only use from<BA256> for Fp25519. I use that in the share conversion from BA256 to Fp25519.